geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStore.scala

```diff
@@ -53,7 +57,11 @@ class KafkaDataStore(val config: KafkaDataStoreConfig)
   override protected def catalog: String = config.catalog
 
   override val metadata: GeoMesaMetadata[String] =
-    new ZookeeperMetadata(s"${config.catalog}/$MetadataPath", config.zookeepers, MetadataStringSerializer)
+    if (config.serialization == SerializationType.CONFLUENT && config.schemaRegistryUrl.isDefined) {
```

Comment:
is there any case where we would want schema registry, but not confluent serialization, or vice versa? maybe we could simplify the config by assuming that a schema registry url implies confluent serialization.

Reply:
Are you suggesting omit SerializationType.CONFLUENT?

Reply:
no, just to ignore the serialization type that is configured if a confluent url is specified, and always use CONFLUENT as the type in that case